api/components/tags/index.yaml

```diff
@@ -84,3 +84,25 @@ on_status:
           - $ref: "./tag-group.yaml#/order/fulfillments/tags/1"
           - $ref: "./tag-group.yaml#/order/fulfillments/tags/2"
       tags: *BAP_BPP_TERMS    
+
+cancel:
+  message:
+    tags:
+      $ref: "./tag-group.yaml#/CUSTOMER_INFO"
```

Comment:
We are actually using `CONSUMER` as a term everywhere to refer to the buyer/investor. Example - `CONSUMER_TNC` tag in `PLAN_INFORMATION` tag group. And also, `cancelled_by` in `cancellation` object in case the order is cancelled by the buyer/investor.

I suggest we use `CONSUMER_INFO` instead of `CUSTOMER_INFO`. What do you think?